src/main/kotlin/com/terraformation/backend/accelerator/db/VoteStore.kt

```diff
@@ -0,0 +1,101 @@
+package com.terraformation.backend.accelerator.db
+
+import com.terraformation.backend.accelerator.model.VoteModel
+import com.terraformation.backend.accelerator.model.toModel
+import com.terraformation.backend.auth.currentUser
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.VoteOption
+import com.terraformation.backend.db.accelerator.tables.daos.ProjectVotesDao
+import com.terraformation.backend.db.accelerator.tables.pojos.ProjectVotesRow
+import com.terraformation.backend.db.accelerator.tables.references.PROJECT_VOTES
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.UserId
+import jakarta.inject.Named
+import java.time.InstantSource
+import org.jooq.Condition
+import org.jooq.DSLContext
+
+@Named
+class VoteStore(
+    private val clock: InstantSource,
+    private val dslContext: DSLContext,
+    private val projectVotesDao: ProjectVotesDao,
+) {
+  fun fetchAllVotesByProject(projectId: ProjectId): List<VoteModel> {
+    return fetch(PROJECT_VOTES.PROJECT_ID.eq(projectId))
+  }
+
+  fun create(model: VoteModel): VoteModel {
+    val now = clock.instant()
+    val currentUserId = currentUser().userId
+
+    val row =
+        ProjectVotesRow(
+            userId = model.userId,
+            projectId = model.projectId,
+            phaseId = model.phase,
+            voteOptionId = model.voteOption,
+            conditionalInfo = model.conditionalInfo,
+            createdBy = currentUserId,
+            createdTime = now,
+            modifiedBy = currentUserId,
+            modifiedTime = now)
```

Comment:
Nit: Maybe put a trailing comma here just to keep future diffs tidier.

Reply:
Done!